packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -308,25 +309,13 @@ const withChunking = (config) => {
     }
 }
 
-// TODO: Once we create a new project for extensibility we'll have the opportunity to better move this utility
-// to a place that can be reused. This util is also used in the `extensions-loader`.
-const normalizeExtensionsList = (extensions = []) =>
-    extensions.map((extension) => {
-        return {
-            packageName: Array.isArray(extension) ? extension[0] : extension,
-            config: Array.isArray(extension) ? {enabled: true, ...extension[1]} : {enabled: true}
-        }
-    })
-
 const staticFolderCopyPlugin = new CopyPlugin({
     patterns: [
         {
             from: 'app/static/',
             to: 'static/'
         },
-        ...normalizeExtensionsList(appConfig?.extensions).map((extension) => {
-            const {packageName} = extension
-            // Parse the extension name out.
+        ...getInstalledExtensions().map((packageName) => {
```

Comment:
For copying the static files, what we really want is to copy from all _installed_ extensions, is that right? Not from the configured extensions only.